src/main/java/org/nixos/idea/settings/NixLangSettingsConfigurable.java

```diff
@@ -0,0 +1,107 @@
+package org.nixos.idea.settings;
+
+import com.intellij.openapi.options.Configurable;
+import com.intellij.openapi.options.ConfigurationException;
+import com.intellij.openapi.options.SearchableConfigurable;
+import com.intellij.openapi.util.NlsContexts;
+import com.intellij.ui.RawCommandLineEditor;
+import com.intellij.ui.TitledSeparator;
+import com.intellij.ui.components.JBCheckBox;
+import com.intellij.ui.components.JBLabel;
+import com.intellij.util.ui.FormBuilder;
+import org.jetbrains.annotations.NonNls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.nixos.idea.lsp.ui.CommandSuggestionsPopup;
+
+import javax.swing.*;
+import java.util.List;
+
+public class NixLangSettingsConfigurable implements SearchableConfigurable, Configurable.Beta {
+    private static final List<CommandSuggestionsPopup.Suggestion> BUILTIN_SUGGESTIONS = List.of(
+            CommandSuggestionsPopup.Suggestion.builtin("<html>Use <b>nixpkgs-fmt</b> from nixpkgs</html>",
+                    "nix --extra-experimental-features \"nix-command flakes\" run nixpkgs#nixpkgs-fmt")
+    );
+
+    private @Nullable JBCheckBox myEnabled;
+    private @Nullable RawCommandLineEditor myCommand;
+    private @Nullable JBLabel myTextArea;
+
+    @Override
+    public @NotNull @NonNls String getId() {
+        return "org.nixos.idea.lsp.NixLangSettingsConfigurable";
+    }
+
+    @Override
+    public @NlsContexts.ConfigurableName String getDisplayName() {
+        return "Nix Language Server (LSP)";
```

Comment:
issue: Inconsistent with display name specified in `plugin.xml`. (Copy & paste issue)